.github/workflows/selenium.yml

```diff
@@ -0,0 +1,69 @@
+# This workflow will run headful selenium tests.
+
+name: Selenium tests
+
+# Declare default permissions as read only.
+permissions: read-all
+
+on:
+  merge_group:
+  pull_request:
+  push:
+    branches: 'main'
+  workflow_dispatch:
+    inputs:
+      verbose:
+        description: Verbose logging
+        default: false
+        required: false
+        type: boolean
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  selenium:
+    strategy:
+      fail-fast: false
+      matrix:
+        # TODO(#876): Add Windows CI.
+        os: [ubuntu-latest] # macos-latest,
```

Comment:
TODO: remove or move commented out code to TODO

Reply:
Done